DailyReportWeb/DependencyInjection.cs

```diff
@@ -23,7 +23,8 @@ public static void Init()
         {
             var builder = new ContainerBuilder();
 
-            builder.RegisterServicesFromAssembly<JiraService>();            
+            builder.RegisterServicesFromAssembly<JiraService>();
+            builder.RegisterServicesFromAssembly<BitBucketService>();
```

Comment:
JiraService and BitBucketService are both in the same assembly. Won't this call registers the same services twice?